internal/ui/confirm_test.go

```
@@ -4,10 +4,36 @@ import (
+func TestPasteClosesConfirmWithoutConfirming(t *testing.T) {
+	m := buildModel(t)
+	createSession(t, m, "keeper", t.TempDir(), "")
+	m.selectSessionRow(t, "keeper")
+	sess, _ := m.selected()
+	m.prepareDelete()
+	if m.mode != modeConfirmDelete {
+		t.Fatalf("mode = %v, want modeConfirmDelete", m.mode)
+	}
+
+	updated, _ := m.Update(tea.PasteMsg{Content: "y\n"})
+	m = updated.(*Model)
```

golangci-lint (2.13.2)

[error] type assertion must be checked

(forcetypeassert)